frontend/src/components/search/error/SearchErrorAlert.tsx

```diff
@@ -0,0 +1,12 @@
+const SearchErrorAlert = () => (
+  <div className="usa-alert usa-alert--error margin-top-2" role="alert">
+    <div className="usa-alert__body">
+      <h4 className="usa-alert__heading">We&apos;re sorry.</h4>
+      <p className="usa-alert__text">
+        There seems to have been an error. Please try your search again.
+      </p>
+    </div>
+  </div>
+);
+
+export default SearchErrorAlert;
```

Comment:
Could take the status code / error type and provide additional info about the error in the future